hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/sink/clustering/ClusteringCommitSink.java

```diff
@@ -150,6 +159,13 @@ private void doCommit(String instant, HoodieClusteringPlan clusteringPlan, List<
     long numErrorRecords = statuses.stream().map(WriteStatus::getTotalErrorRecords).reduce(Long::sum).orElse(0L);
 
     if (numErrorRecords > 0 && !this.conf.getBoolean(FlinkOptions.IGNORE_FAILED)) {
+      events.forEach(event -> {
+        Option<Long> failRecordCnt = Option.fromJavaOptional(
+                event.getWriteStatuses().stream().map(WriteStatus::getTotalErrorRecords).reduce(Long::sum));
+        if (failRecordCnt.isPresent() && failRecordCnt.get() > 0) {
+          LOG.warn("Clustering event with failed record num: " + failRecordCnt + " from task " + event.getTaskID());
```

Comment:
`HoodieCreateHandle#write` logs the failed record, therefore does this need to log failed record?

Reply:
Purpose is to find task id that contain failed writes, and log the failed number by the way.

Reply:
The write task alreay log some messages before it sends the events, does that make sense to you?

Reply:
If we have 100 task manger to run compact/clustering, maybe hard to find which tm contains the error log?

Reply:
Or maybe we can just log a message while receiving the event.

Reply:
good idea, done, please take another look